src/init.cpp

```diff
@@ -342,6 +342,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-dbcache=<n>", strprintf(_("Set database cache size in megabytes (%d to %d, default: %d)"), nMinDbCache, nMaxDbCache, nDefaultDbCache));
     strUsage += HelpMessageOpt("-debuglogfile=<file>", strprintf(_("Specify location of debug log file: this can be an absolute path or a path relative to the data directory (default: %s)"), DEFAULT_DEBUGLOGFILE));
     strUsage += HelpMessageOpt("-exportdir=<dir>", _("Specify directory to be used when exporting data"));
+    strUsage += HelpMessageOpt("-ibdskiptxverification", _("Skip Tx verification during initial block download"));
```

Comment:
```suggestion
    strUsage += HelpMessageOpt("-ibdskiptxverification", _("Skip Tx verification during initial block download up to last checkpoint height. Requires -checkpoints."));
```

Reply:
Should `-ibdskiptxverification` imply checkpoints, rather than require it be specified separately?

Reply:
@str4d how about this: https://github.com/zcash/zcash/pull/4770/files#diff-c865a8939105e6350a50af02766291b7R968